tests/nightly/test_large_array.py

```diff
@@ -185,7 +186,19 @@ def test_pick():
     b = mx.nd.ones(shape=(256*35,))
     res = mx.nd.pick(a,b)
     assert res.shape == b.shape
-
+    
+def test_depthtospace():
+    shape_inp = (LARGE_X, 8, 4, 2)
+    data = rand_ndarray(shape_inp, 'default')
+    data_np = data.asnumpy()
+    output = mx.nd.depth_to_space(data, 2)
+
```

Comment:
add extra line to follow pep8 style guide

Reply:
done